### PR TITLE
Removing Unity

### PR DESCRIPTION
**Pull request by HunterBarclay**
### Description
Completely deleting the Unity project, as well as supporting files and components

### Removed
- [x] Removed `/engine/`
- [x] Removed `/api/`
- [x] Removed `/scripts/`
- [x] Removed engine workflows
- [x] Removed api workflows

### Updated
- [x] Updated workflow names
- [x] Updated `README.md` (Temporary)
- [x] Updated Logo storage location

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,111 +1,72 @@
...
+- **Exploring the Field Environment**: Every year on kickoff, for both FTC and FRC FIRST® competitions, Synthesis has the newest field available immediately. This allows teams to explore the field through a 3D model, drive a robot around, and begin to strategize for the upcoming season's game.
 - **Driver Practice & Strategy**: Not getting enough driver practice or don't have a full field available to you? Synthesis has you covered with the ability to drive your robot around with a gamepad from a first-person view at the driver station; allowing you to get a feel for potential control scheme layouts and any line-of-sight challenges that may arise. This also allows the drive team and the programmers to communicate about what control layouts work best for each driver.
 
 ## Getting Started
 
 If you are a FIRST robotics student who just wants to use Synthesis, you *don't* need this repo. Simply **install the latest release of Synthesis from [synthesis.autodesk.com/download](https://synthesis.autodesk.com/download.html)**.
 
+[!IMPORTANT]
+Moving to [synthesis.autodesk.com]
```
Comment (BrandonPacewic) [inline README.md]:
```suggestion
> [!IMPORTANT]
> Moving to [synthesis.autodesk.com]
```

The important tag needs to be formatted in this way to appear properly.

File: .github/workflows/BlackFormat.yml [hunk 2]
```diff
@@ -1,4 +1,4 @@
-name: Format Validation (Black)
+name: Fusion - Format Validation
```
Comment (BrandonPacewic) [inline .github/workflows/BlackFormat.yml]:
```suggestion
name: Fusion - Black Format Validation
```

Tiny nitpick, should be consistent with other formatting workflow.

https://github.com/Autodesk/synthesis/blob/00bb1addcffae3eb14e7c00ae895d7cf48943232/.github/workflows/FissionESLintFormat.yml#L1

**Review by BrandonPacewic — CHANGES_REQUESTED** [review]
I believe the `init.bat` and `init.sh` scripts at root need to be updated as well.

File: .github/workflows/BlackFormat.yml (hunk 2 above)
Comment (HunterBarclay) [reply .github/workflows/BlackFormat.yml]: Right, I thought we didn't called it ES Lint.

**Comment by HunterBarclay** [comment]
@BrandonPacewic Made the changes plus changing the header for the "other components" section of the README. I'm leaving in the init scripts for now. Once we decide if we're going to use the protocols folder, we'll remove/keep them. For now, I just removed all the setup for things we removed.

**Review by BrandonPacewic — APPROVED** [review]

**Review by a-crowell — APPROVED** [review]
Excited to see this cleaned out. Almost 600,000 lines, wow.

**Pull request merged**